detail/src/main/kotlin/io/goooler/demoapp/detail/api/DetailApi.kt

```diff
@@ -6,9 +6,8 @@ import retrofit2.http.Path
 
 interface DetailApi {
 
-  @GET("https://api.github.com/repos/{owner}/{repo}")
+  @GET("https://api.github.com/repos/{fullName}")
   suspend fun getRepoDetail(
-    @Path("owner") owner: String,
-    @Path("repo") repo: String
+    @Path("fullName") fullName: String
```

Comment:
```
URL: https://api.github.com/repos/Goooler/DemoApp
Method: GET
Protocol: h2
Status: Complete
Response: 404 
SSL: Yes

Request time: Thu Mar 17 01:08:51 GMT+08:00 2022
Response time: Thu Mar 17 01:08:51 GMT+08:00 2022
Duration: 417 ms

Request size: 0 B
Response size: 91 B
Total size: 91 B

---------- Request ----------

flavor: dev
versionName: 1.5.0
versionCode: 105000
debug: true
Host: api.github.com
Connection: Keep-Alive
Accept-Encoding: gzip
User-Agent: okhttp/4.9.3
If-Modified-Since: Wed, 16 Mar 2022 16:56:52 GMT

(body is empty)

---------- Response ----------

server: GitHub.com
date: Wed, 16 Mar 2022 17:08:52 GMT
content-type: application/json; charset=utf-8
x-github-media-type: github.v3; format=json
access-control-expose-headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
access-control-allow-origin: *
strict-transport-security: max-age=31536000; includeSubdomains; preload
x-frame-options: deny
x-content-type-options: nosniff
x-xss-protection: 0
referrer-policy: origin-when-cross-origin, strict-origin-when-cross-origin
content-security-policy: default-src 'none'
vary: Accept-Encoding, Accept, X-Requested-With
content-encoding: gzip
x-ratelimit-limit: 60
x-ratelimit-remaining: 50
x-ratelimit-reset: 1647451801
x-ratelimit-resource: core
x-ratelimit-used: 10
content-length: 91
x-github-request-id: D9C0:721A:33E3D6:388209:623219A1

{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest"
}
```

Reply:
```
URL: https://api.github.com/repos/Goooler/DemoApp
Method: GET
Protocol: h2
Status: Complete
Response: 304 
SSL: Yes

Request time: Thu Mar 17 01:12:50 GMT+08:00 2022
Response time: Thu Mar 17 01:12:50 GMT+08:00 2022
Duration: 461 ms

Request size: 0 B
Response size: null
Total size: 0 B

---------- Request ----------

flavor: dev
versionName: 1.5.0
versionCode: 105000
debug: true
Host: api.github.com
Connection: Keep-Alive
Accept-Encoding: gzip
User-Agent: okhttp/4.9.3
If-None-Match: W/"ed644373a5619886b2d794ee1ed7ba053d4827fa489aea8aadabd07b08d89446"

(body is empty)

---------- Response ----------

server: GitHub.com
date: Wed, 16 Mar 2022 17:12:51 GMT
content-type: application/json; charset=utf-8
cache-control: public, max-age=60, s-maxage=60
vary: Accept, Accept-Encoding, Accept, X-Requested-With
etag: W/"ed644373a5619886b2d794ee1ed7ba053d4827fa489aea8aadabd07b08d89446"
last-modified: Mon, 07 Mar 2022 06:09:46 GMT
x-github-media-type: github.v3; format=json
access-control-expose-headers: ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset
access-control-allow-origin: *
strict-transport-security: max-age=31536000; includeSubdomains; preload
x-frame-options: deny
x-content-type-options: nosniff
x-xss-protection: 0
referrer-policy: origin-when-cross-origin, strict-origin-when-cross-origin
content-security-policy: default-src 'none'
content-encoding: gzip
x-ratelimit-limit: 60
x-ratelimit-remaining: 48
x-ratelimit-reset: 1647451800
x-ratelimit-resource: core
x-ratelimit-used: 12
x-github-request-id: 965C:1B7C:2A24CE:2EBF4D:62321A90

(body is empty)
```